lib/apiservers/engine/backends/container.go

```diff
@@ -58,6 +58,19 @@ import (
 
 const (
 	bridgeIfaceName = "bridge"
+
```

Comment:
No, not really, it was just meant to be a starting point. However, when I tried with 64MB to run a busybox `/bin/top` container, it wasn't enough. I tried 256MB and it kernel panicked. I'm messing around now trying to figure out a minimal value that makes sense. I'll report back soon. In the meantime, given how ESX handles memory overcommitment, it seems we have a good amount of wiggle room, but I'm not really sure what should be considered "too much"...